modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -222,7 +222,14 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
         $this->addBasicText(
             'DCCID', 'DCCID', array('size' => 10, "maxlength" => 25)
         );
-        $this->addSelect('ProjectID', 'Project', $allAr + Utility::getProjectList());
+
+        $config =& NDB_Config::singleton();
+        $projAr = $allAr; 
+        if ($config->getSetting('useProjects') === "true") {
+            $projAr += Utility::getProjectList(); 
+        }
+        $this->addSelect('ProjectID', 'Project', $projAr);  
```

Comment:
the addSelect should be inside the if statement. Also do you need to add check in the template file?